### PR TITLE
Rearrange navigator

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -52,33 +52,39 @@ strict: true
...
-          - Observability Overview: observability/observability-overview.md
-          - Perform Root Cause Analysis: observability/perform-root-cause-analysis.md
-          - Performance Analysis: observability/analyze-performance.md
-    - Insights:
+    - Concepts:
+          - Concepts: concepts/concepts.md
+    - Quick Start Guide:
+          - Quick Start Guide: qsg/quick-start-guide.md
+    - Develop:
+          - Create Project
```
Comment (shaniR) [inline en/mkdocs.yml:60]: We didn't have this noh?

File: en/mkdocs.yml [hunk 2]
```diff
@@ -52,33 +52,39 @@ strict: true
...
+                - Create Integration: develop/create-integration.md
+    - Deploy:
+          - Deploy APIs: deploy/deploy-apis.md
+          - CI/CD: deploy/ci-cd.md
+    - Manage:
+          - Create API Revisions: manage/create-api-revisions.md
+          - Add Custom Properties: manage/add-custom-properties.md
+          - Create API Product: manage/create-api-product.md
+          - Manage Endpoints: manage/manage-endpoints.md
+          - API Documentation: manage/api-documentation.md
```
Comment (shaniR) [inline en/mkdocs.yml:71]: Should the API related ones come here?

File: en/mkdocs.yml [hunk 3]
```diff
@@ -52,33 +52,39 @@ strict: true
...
+        - Observability Overview: observe/observability-overview.md
+        - Perform Root Cause Analysis: observe/perform-root-cause-analysis.md
+    - Derive Insights: insights/derive-insights.md
           - Derive Insights: insights/derive-insights.md
           - Configure Alerts: insights/configure-alerts.md
           - Choreo Insights API:
                 - Overview: insights/choreo-insights-api.md
                 - Explorer: insights/graphiql-explorer.md
     - Reference:
           - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
```
Comment (shaniR) [inline en/mkdocs.yml:86]: These would move to the integrations section

File: en/mkdocs.yml [hunk 4]
```diff
@@ -52,33 +52,39 @@ strict: true
...
-          - Anonymous Access: references/anonymous-access.md
           - Data Mapping: references/data-mapping.md
```
Comment (shaniR) [inline en/mkdocs.yml:87]: This and connectors too

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/mkdocs.yml (hunk 3 above)
Comment (RukshiW) [reply en/mkdocs.yml:82]: Moved Perform Root Cause Analysis.

File: en/mkdocs.yml (hunk 2 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Removed the ones that do not belong here as per our offline chat.

File: en/mkdocs.yml (hunk 1 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Removed it.

File: en/mkdocs.yml (hunk 3 above)
Comment (shaniR) [reply en/mkdocs.yml:82]: um... was referring to the topics in the Reference section.

File: en/mkdocs.yml [hunk 5]
```diff
@@ -52,34 +52,38 @@ strict: true
...
-          - Integration Concepts: integrations/integration-concepts.md
-          - Build Your First Integration: integrations/build-your-first-integration.md
-    - Remote Apps:
-          - Connect Remote Applications: remote-apps/connect-remote-applications.md
-    - Observability:
-          - Observability Overview: observability/observability-overview.md
-          - Perform Root Cause Analysis: observability/perform-root-cause-analysis.md
-          - Performance Analysis: observability/analyze-performance.md
-    - Insights:
+    - Concepts:
```
Comment (shaniR) [inline en/mkdocs.yml:55]: Shall we add the .md file here? without having a nested level with the same heading?

File: en/mkdocs.yml [hunk 6]
```diff
@@ -52,34 +52,38 @@ strict: true
...
+          - Concepts: concepts/concepts.md
+    - Quick Start Guide:
```
Comment (shaniR) [inline en/mkdocs.yml:57]: same here

File: en/mkdocs.yml [hunk 7]
```diff
@@ -52,34 +52,38 @@ strict: true
...
+          - Quick Start Guide: qsg/quick-start-guide.md
+    - Develop:
+          - Create API: develop/create-api.md
```
Comment (shaniR) [inline en/mkdocs.yml:60]: APIs

File: en/mkdocs.yml [hunk 8]
```diff
@@ -52,34 +52,38 @@ strict: true
...
+          - Create Integration:
```
Comment (shaniR) [inline en/mkdocs.yml:61]: Integrations

File: en/mkdocs.yml [hunk 9]
```diff
@@ -52,34 +52,38 @@ strict: true
...
+                - Data Mapping: references/data-mapping.md
+                - Connectors : references/connectors.md
+    - Deploy:
+          - Deploy APIs: deploy/deploy-apis.md
+          - CI/CD: deploy/ci-cd.md
+    - Manage:
+          - Create API Revisions: manage/create-api-revisions.md
+          - Add Custom Properties: manage/add-custom-properties.md
+          - API Collaboration: manage/api-collaboratio.md
+          - Rate Limiting: manage/rate-limiting.md
```
Comment (shaniR) [inline en/mkdocs.yml:73]: This should go to the API-M section

File: en/mkdocs.yml [hunk 10]
```diff
@@ -52,34 +52,38 @@ strict: true
...
+                - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
+                - Data Mapping: references/data-mapping.md
+                - Connectors : references/connectors.md
+    - Deploy:
+          - Deploy APIs: deploy/deploy-apis.md
+          - CI/CD: deploy/ci-cd.md
+    - Manage:
+          - Create API Revisions: manage/create-api-revisions.md
+          - Add Custom Properties: manage/add-custom-properties.md
+          - API Collaboration: manage/api-collaboratio.md
```
Comment (shaniR) [inline en/mkdocs.yml:72]: This should go to the API-M section but let's remove this.

File: en/mkdocs.yml [hunk 11]
```diff
@@ -52,34 +52,38 @@ strict: true
...
+                - Create Integration: develop/create-integration.md
+                - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
+                - Data Mapping: references/data-mapping.md
+                - Connectors : references/connectors.md
+    - Deploy:
+          - Deploy APIs: deploy/deploy-apis.md
+          - CI/CD: deploy/ci-cd.md
+    - Manage:
+          - Create API Revisions: manage/create-api-revisions.md
+          - Add Custom Properties: manage/add-custom-properties.md
```
Comment (shaniR) [inline en/mkdocs.yml:71]: This should come as a part of API creation

File: en/mkdocs.yml [hunk 12]
```diff
@@ -52,34 +52,38 @@ strict: true
...
+          - Create API: develop/create-api.md
+          - Create Integration:
+                - Create Integration: develop/create-integration.md
+                - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
+                - Data Mapping: references/data-mapping.md
+                - Connectors : references/connectors.md
+    - Deploy:
+          - Deploy APIs: deploy/deploy-apis.md
+          - CI/CD: deploy/ci-cd.md
+    - Manage:
```
Comment (shaniR) [inline en/mkdocs.yml:69]: To this section let's just add a general page.

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/mkdocs.yml (hunk 5 above)
Comment (shaniR) [inline en/mkdocs.yml:55]: In this file, let's add a short description of what we are going to  include in the section. Let's follow the same approach in all sections.

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/mkdocs.yml [hunk 13]
```diff
@@ -52,34 +52,38 @@ strict: true
...
+          - Create API Revisions: manage/create-api-revisions.md
+          - Add Custom Properties: manage/add-custom-properties.md
+          - API Collaboration: manage/api-collaboratio.md
+          - Rate Limiting: manage/rate-limiting.md
+          - API Monetization: manage/api-monetization.md
```
Comment (Nashaath) [inline en/mkdocs.yml:74]: We can avoid having monetization at the moment right?

File: en/mkdocs.yml [hunk 14]
```diff
@@ -52,34 +52,38 @@ strict: true
...
                 - Overview: insights/choreo-insights-api.md
                 - Explorer: insights/graphiql-explorer.md
     - Reference:
-          - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
-          - Anonymous Access: references/anonymous-access.md
-          - Data Mapping: references/data-mapping.md
-          - Test Your Service via Postman: references/test-your-service-via-postman.md
-          - API Path: references/api-path.md
-          - Connectors : references/connectors.md
+          - FAQ: faq.md
```
Comment (Nashaath) [inline en/mkdocs.yml:85]: If FAQ seems like the only item coming under reference for now, shall we have FAQ as the high-level topic. Might be to see that at a glance without having it as a nested topic. We can. introduce Reference if we have more items coming under it. WDYT?

Comment (RukshiW) [reply en/mkdocs.yml]: Done

File: en/mkdocs.yml (hunk 13 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Removed the APIM topics

File: en/mkdocs.yml (hunk 5 above)
Comment (RukshiW) [reply en/mkdocs.yml]: @Nashaath did you mention that you wrote something for this page?

File: en/mkdocs.yml (hunk 12 above)
Comment (RukshiW) [reply en/mkdocs.yml:67]: Done

File: en/mkdocs.yml (hunk 11 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Added just one topic here for now.

File: en/mkdocs.yml (hunk 10 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Removed

File: en/mkdocs.yml (hunk 9 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Removed this

File: en/mkdocs.yml (hunk 8 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Done

File: en/mkdocs.yml (hunk 7 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Done

File: en/mkdocs.yml (hunk 6 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Done

File: en/mkdocs.yml (hunk 5 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Done

File: en/mkdocs.yml [hunk 15]
```diff
@@ -52,34 +52,31 @@ strict: true
...
-          - Connect Remote Applications: remote-apps/connect-remote-applications.md
-    - Observability:
-          - Observability Overview: observability/observability-overview.md
-          - Perform Root Cause Analysis: observability/perform-root-cause-analysis.md
-          - Performance Analysis: observability/analyze-performance.md
-    - Insights:
+    - Concepts: concepts/concepts.md
+    - Quick Start Guide: qsg/quick-start-guide.md
+    - Develop:
+          - Create APIs: develop/create-api.md
```
Comment (shaniR) [inline en/mkdocs.yml:58]: Shall we just name this APIs

File: en/mkdocs.yml [hunk 16]
```diff
@@ -52,34 +52,31 @@ strict: true
...
+          - Create Integrations:
```
Comment (shaniR) [inline en/mkdocs.yml:59]: and this Integrations

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/mkdocs.yml (hunk 5 above)
Comment (Nashaath) [reply en/mkdocs.yml]: I suggested sample content for the home page.

**Pull request merged**